docs/content/guide/animations.ngdoc

```diff
@@ -59,191 +65,190 @@ You may also want to setup a separate CSS file for defining CSS-based animations
+(**for simplicity, we rely on jQuery to perform animations here**):
 
 ```js
 myModule.animation('.repeated-item', function() {
   return {
     enter: function(element, done) {
-      element.css('opacity',0);
-      jQuery(element).animate({
-        opacity: 1
-      }, done);
-
-      // optional onDone or onCancel callback
-      // function to handle any post-animation
-      // cleanup operations
+      // Initialize the element's opacity
+      element.css('opacity', 0);
+
+      // Animate the element's opacity
+      // (`element.animate()` is provided by jQuery)
+      element.animate({opacity: 1}, done);
+
+      // Optional `onDone`/`onCancel` callback function
+      // to handle any post-animation cleanup operations
       return function(isCancelled) {
-        if(isCancelled) {
-          jQuery(element).stop();
+        if (isCancelled) {
+          // Abort the animation if cancelled
+          // (`element.stop()` is provided by jQuery)
+          element.stop();
         }
-      }
+      };
     },
     leave: function(element, done) {
+      // Initialize the element's opacity
       element.css('opacity', 1);
-      jQuery(element).animate({
-        opacity: 0
-      }, done);
 
-      // optional onDone or onCancel callback
-      // function to handle any post-animation
-      // cleanup operations
-      return function(isCancelled) {
-        if(isCancelled) {
-          jQuery(element).stop();
-        }
-      }
-    },
-    move: function(element, done) {
-      element.css('opacity', 0);
-      jQuery(element).animate({
-        opacity: 1
-      }, done);
+      // Animate the element's opacity
+      // (`element.animate()` is provided by jQuery)
+      element.animate({opacity: 0}, done);
 
-      // optional onDone or onCancel callback
-      // function to handle any post-animation
-      // cleanup operations
+      // Optional `onDone`/`onCancel` callback function
+      // to handle any post-animation cleanup operations
       return function(isCancelled) {
-        if(isCancelled) {
-          jQuery(element).stop();
+        if (isCancelled) {
+          // Abort the animation if cancelled
+          // (`element.stop()` is provided by jQuery)
+          element.stop();
         }
-      }
+      };
     },
 
-    // you can also capture these animation events
+    // We can also capture the following animation events:
+    move: function(element, done) {},
     addClass: function(element, className, done) {},
     removeClass: function(element, className, done) {}
   }
 });
 ```
 
 With these generated CSS class names present on the element at the time, AngularJS automatically
-figures out whether to perform a CSS and/or JavaScript animation. If both CSS and JavaScript animation
-code is present, and match the CSS class name on the element, then AngularJS will run both animations at the same time.
+figures out whether to perform a CSS and/or JavaScript animation. If both CSS and JavaScript
```

Comment:
JS animations - because you can run CSS in JS: http://plnkr.co/edit/rdRAZTiXHNAFTEOHyZVo?p=preview

Although I'm surprised this isn't explicitly said in the migrate docs. But it is here: https://code.angularjs.org/snapshot/docs/api/ngAnimate

Unfortuante spreading out of content ... moving this in guide would be a good passion project for rainy afternoons. :p